src/RPCServer.ts

```diff
@@ -81,12 +81,12 @@ class RPCServer {
    * transform streams that convert `Uint8Array` to `JSONRPCRequest` on the forward
    * path and `JSONRPCResponse` to `Uint8Array` on the reverse path.
    * @param obj.streamKeepAliveTimeoutTime - Time before a connection is cleaned up due to no activity. This is the
-   * value used if the handler doesn't specify its own timeout time. This timeout is advisory and only results in a
+   * value used if the handler doesn't specify its own handlerTimeout time. This handlerTimeout is advisory and only results in a
```

Comment:
You could call it the `Handler.timeout`.

Reply:
timeout gets confusing in a larger code base with so many timeouts, specific name is better

Reply:
It's already namespaced under `Handler`.

Reply:
I prefer keeping the name as is, as per the context of the `Handler` class.

Reply:
Name changed back to timeout